cmd/kube-controller-manager/app/core.go

```diff
@@ -271,7 +271,8 @@ func startPersistentVolumeBinderController(ctx context.Context, controllerContex
 		EnableDynamicProvisioning: controllerContext.ComponentConfig.PersistentVolumeBinderController.VolumeConfiguration.EnableDynamicProvisioning,
 		FilteredDialOptions:       filteredDialOptions,
 	}
-	volumeController, volumeControllerErr := persistentvolumecontroller.NewController(params)
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "persistent volume controller"))
```

Comment:
Nit: `persistentvolume-binder-controller`, see https://docs.google.com/spreadsheets/d/1s0Y3ruLDUWR8k0KyMLlY-5gm09TDiOjONFsMqs0-Fjw/edit?pli=1#gid=0 /cc @atiratree since you'll be touching on those in https://github.com/kubernetes/kubernetes/pull/115813

Reply:
done